docs/modules/data-structures/pages/listening-for-map-entries.adoc

```diff
@@ -94,8 +105,24 @@ C++::
 --
 [source,cpp]
 ----
-
-
+#include "Employee.h"
+
+int main() {
+    hazelcast::client::client_config config;
+    config.set_property("hazelcast.map.entry.filtering.natural.event.types", "true");
```

Comment:
there is no such property. also, we should also remove that for Java, it should not be needed.